src/pug/includes/head.pug

```diff
@@ -10,9 +10,5 @@ meta(name="description" content="swag opportunities for developers")
font-display:swap}:root,body,html{width:100%;height:100%;background:#fff;font-family:Cabin,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Cantarell,'Open Sans','Helvetica Neue',sans-serif;font-size:16px;text-rendering:optimizeLegibility;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;-webkit-overflow-scrolling:touch}*{padding:0;margin:0;box-sizing:border-box;text-rendering:optimizeLegibility}.flex{display:flex;align-items:center;justify-content:center}.container{width:100%;height:100%;justify-content:center;position:relative}.container{min-height:100vh;width:100vw;overflow-x:hidden}.hidden{display:none}section.main{width:90%;flex-direction:column;margin:0 auto}section.main .heading{flex-direction:column;min-height:40vh;text-align:center;margin:6vh 2vh}section.main .heading h1{font-size:3rem;color:#111;width:90%}section.main .heading p{font-size:1.2rem;color:#555;margin-top:20px;width:90%}section.main .buttons{justify-content:flex-start;padding-bottom:15px;max-width:1150px;width:100%;flex-direction:column;align-items:flex-start}section.main .buttons .top{flex-wrap:wrap;width:100%;justify-content:flex-start}section.main .buttons .item{padding:5px}section.main .buttons .select-wrapper{position:relative}section.main .buttons .select-wrapper:before{position:absolute;top:50%;right:10px;width:0;height:0;content:'';transform:rotate(0) translate3d(0,-50%,0);border-width:4px 4px 0;border-style:solid;border-color:#6c7a86 transparent transparent}section.main .buttons select{margin:0 5px;background:#fff;border:none;border-bottom:2px solid #bbb;border-radius:0;padding:5px 20px 5px 5px;-webkit-appearance:none;-moz-appearance:none;font-family:Cabin,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Cantarell,'Open Sans','Helvetica Neue',sans-serif;font-size:.9rem}section.main .content{max-width:1100px;flex-wrap:wrap;align-items:flex-start;justify-content:space-around;margin:0 -10px;width:100%}section.main .content .item{width:100%;max-width:350px;padding:30px 10px;border-top:1px solid #ddd;display:none}@media only screen and (max-width:500px){section.main .content .item{max-width:none}}section.main .content .item .title{justify-content:space-between}section.main .content .item .title h1{font-size:2rem}section.main .content .item .title .difficulty{height:30px;width:30px;border-radius:50%;border:2px solid #000}section.main .content .item .title .difficulty.hard{background:#ff0009}section.main .content .item p{font-size:1.2rem;line-height:1.4;color:#222}section.main .content .item p.swag{margin:5px -3px 10px -3px}section.main .content .item p.swag span{background:#737373;color:#fff;border-radius:3px;margin:0 3px;padding:2px 5px;font-size:1rem}section.main .content .item img{height:100%;width:auto}section.main .content .item .img-container{height:300px;width:100%;overflow:hidden;padding-bottom:15px}.sr-only{position:absolute!important;clip:rect(1px 1px 1px 1px);clip:rect(1px,1px,1px,1px)}#content .item.visible{display:initial}
```

Comment:
This CSS is being generated by gulp so the way to do that would be passing it to the pug template.
I think we could keep our workflow for our CSS but maybe  lazyload the selectr CSS.

Reply:
This is the critical path CSS generated by a third party site

Reply:
We might be able to extract our CSS critical path with https://github.com/addyosmani/critical within our gulp pipeline.

Reply:
Critical CSS is normally a great idea, but given we're using H2 Multiplexing and the `index-{hash}.css` file is 2k (ish), I don't think it's really necessary. This would be a premature optimization. Looking at the lighthouse report, we would get exponentially better performance gains by deferring the loading of offscreen images. We could definitely defer loading the selectr css though